### ISSUE TITLE
Team leader interviews 

### ISSUE THREAD

**Issue by emreerhan** [hackseq/hackseqRNA#21]
Hi all,

Please sign up for 2 team leader interviews [here](https://docs.google.com/spreadsheets/d/1339hSbSsj-XMii389BqMARCiE5BShd9JWL5lvwmT5s4/edit?usp=sharing)

You can find a list of interview questions [here](https://docs.google.com/document/d/1ewRni5zDGPv64W11twRjulx11Pt7TcvIekKZczXE1sk/edit?usp=sharing)

At the end of the interview, please request the information found in [this ticket](https://github.com/hackseq/hackseqRNA/issues/20) with a deadline of **March 23**.

**Comment by emreerhan** [comment]
@hackseq/hackseqrna_org just pinging all of you to sign up for team leader interviews. If you are a new organizer and you would like support from a past organizer, sign up for an interview that already has a past organizer signed up.

**Comment by emreerhan** [comment]
@hackseq/hackseqrna_org can everyone proceed with emailing their interviewees to confirm an interview time?

**Comment by ababaian** [comment]
I think a lot of them have responded with available times already. Do we have this information available in a table?

**Comment by emreerhan** [comment]
Yes. The info is all in the team leader interview spreadsheet on the drive.

**Comment by emreerhan** [comment]
There are a few empty cells. Those TLs haven't responded yet.

**Comment by ababaian** [comment]
I accidentally a column, I see it now.

**Comment by ababaian** [comment]
Also when you're emailing can I suggest to give them information here #20 in your emails.

A

**Comment by ababaian** [comment]
How does this as a draft email sound @emreerhan?

> Good afternoon \< TEAM LEADER \>,
> 
> We are Artem and Jenny, we're hackseqRNA organizers and would like to have a brief (~30-45 minute) chat about your proposed project.
> 
> From your availability the time which would work best for us is March 19th, at 10AM PST (Noon CST). Can you confirm this is OK?
> 
> The objective of this conversation is to help refine your project to optimally fit within the scope of a hackathon. You can get a brief sense of what we'd like to discuss by reviewing our main interview questions here (https://docs.google.com/document/d/1ewRni5zDGPv64W11twRjulx11Pt7TcvIekKZczXE1sk/edit?usp=sharing. In addition we'd like to get a sense of what you're hoping to accomplish and how we as organizers can help you make that a success.
> 
> Following the discussion we'd like to request some additional information to help us 'advertise' your project for participants and successfully recruit the correct people. You can find that information on page two of the above link.
> 
> Finally, we will discuss hackseqRNA contingencies in light of the COVID-19 pandemic and how we can best proceed.
> 
> Let us know if you have any questions!
> 
> Cheers,
> 
> Artem + Jenny
> On behalf of the hackseqRNA organizing committee

**Comment by emreerhan** [comment]
That looks great!
